format/src/main/thrift/schema.thrift

```diff
@@ -192,13 +192,13 @@ struct DataMapSchema  {
     1: required string dataMapName;
     // class name
     2: required string className;
-    // relation indentifier
-    3: optional RelationIdentifier relationIdentifire;
-    // in case of preaggregate it will be used to maintain the child schema
-    // which will be usefull in case of query and data load
-    4: optional TableSchema childTableSchema;
     // to maintain properties like select query, query type like groupby, join
```

Comment:
change description, it stores property specified in DMPROPERTIES when creating datamap

Reply:
ok